chapters/conclusion/README.md

```diff
@@ -37,7 +37,8 @@ To Jennifer, who always supports my crazy ideas.
 To the following contributors who improved the Little ASP.NET Core Book:
 
 * 0xNF
-* Matt Welke
+* Matt Welke (welke)
```

Comment:
username in brackets? fyi, mine is "welkie" :P

Reply:
D'oh, fixed.